bindings.cpp

```diff
@@ -1077,8 +1077,16 @@ bool DeRestPluginPrivate::sendConfigureReportingRequest(BindingTask &bt)
             rq4.minInterval = 1;
             rq4.maxInterval = 600;
             rq4.reportableChange16bit = 0xffff;
+            
+            ConfigureReportingRequest rq5;
+            rq5.dataType = deCONZ::Zcl8BitEnum;
+            rq5.attributeId = 0x001C;        // Thermostat mode
+            rq5.minInterval = 1;
+            rq5.maxInterval = 600;
+            rq5.reportableChange16bit = 0xffff;
 
-            return sendConfigureReportingRequest(bt, {rq, rq2, rq3, rq4});
+            return sendConfigureReportingRequest(bt, {rq, rq2, rq3, rq4}) ||
```

Comment:
Not sure about the logic behind this. Is the first `sendConfigureReportingRequest()` supposed to fail? Because `sendConfigureReportingRequest(bt, {rq5})` will only be executed if the former part fails.